compose/config/config.py

```diff
@@ -186,11 +186,6 @@ def version(self):
         if version == '3':
             version = V3_0
 
```

Comment:
I'm surprised that there is no parsing for 3.1version here

        if version == '3.1':
            version = V3_1
no?